server/src/main/java/org/elasticsearch/rest/BaseRestHandler.java

```diff
@@ -76,13 +76,18 @@ public final long getUsageCount() {
     @Override
     public abstract List<Route> routes();
 
+    private static final Set<String> ALWAYS_SUPPORTED = Set.of("format", "Accept", "filter_path", "pretty", "human");
```

Comment:
Is `Accept` a query param? Seems like a header

Reply:
Doh, it's a `header` mixed in with `params`